components/microsoft_onedrive/actions/upload-file/upload-file.mjs

```diff
@@ -23,30 +22,14 @@ export default {
     filePath: {
       type: "string",
       label: "File Path",
-      description: "The path to the image file saved to the `/tmp` directory (e.g. `/tmp/image.png`). [See the documentation](https://pipedream.com/docs/workflows/steps/code/nodejs/working-with-files/#the-tmp-directory).",
+      description: "The path to the file saved to the `/tmp` directory (e.g. `/tmp/image.png`). [See the documentation](https://pipedream.com/docs/workflows/steps/code/nodejs/working-with-files/#the-tmp-directory).",
     },
     filename: {
       type: "string",
       label: "Name",
       description: "Name of the new uploaded file",
     },
   },
-  methods: {
```

Comment:
I think the run method needs to be updated to reflect this, i.e..
`this.onedrive.uploadFile` instead of `this.uploadFile`